src/types/chains-config/ControllerConfig.ts

```diff
@@ -25,3 +29,33 @@ export interface ControllerConfig {
 		finalizes: boolean;
 	};
 }
+
+/**
+ * Used in BlocksService, to persist decorated metadata to avoid expensive calls
```

Comment:
Change to something like `Cache for runtime version specific metadata constants.`